sliceanddice/host/build.gradle

```diff
@@ -0,0 +1,24 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileSdkVersion 'android-P'
+
+    defaultConfig {
+        applicationId 'com.novoda.sliceanddice'
+        minSdkVersion 24
+        targetSdkVersion 28
```

Comment:
Massive hack: by putting the `targetSdkVersion` to 28 instead of `'P'` we can get the app to install on non-P devices. It's not necessarily going to work as of 28.0.0-alpha1 but since the `minSdkVersion` is `24` for the slices compat libraries, it should stay this way.

Doing the same thing on the `provider` module.

Reply:
It's an uber hack, not even sure it's _meant_ to work, but it does, so... 🤷‍♂️